doc-content/shared-kie-docs/src/main/asciidoc/Workbench/ArchetypeManagement/proc-business-central-listing-archetypes.adoc

```diff
@@ -9,6 +9,6 @@ The *Archetypes* page lists all the archetypes that are added in {CENTRAL}. This
 
 .Procedure
 
-. In {CENTRAL}, select the *Admin* icon in the top-right corner of the screen and select *Archetypes*.
+. In {CENTRAL}, select the *Admin* icon in the upper-right corner of the screen and select *Archetypes*.
```

Comment:
See [Style Guide - Location of interface elements](https://source.redhat.com/groups/public/ccs/ccs_folder/ibm_style_guide/ibm_style_062121_q2_2021pdf) comment above.

Reply:
Done